backend/src/main/java/gov/cdc/usds/simplereport/service/dataloader/PatientLinkDataLoader.java

```diff
@@ -0,0 +1,53 @@
+package gov.cdc.usds.simplereport.service.dataloader;
+
+import gov.cdc.usds.simplereport.db.model.PatientLink;
+import gov.cdc.usds.simplereport.db.repository.PatientLinkRepository;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PatientLinkDataLoader extends KeyedDataLoaderFactory<UUID, PatientLink> {
```

Comment:
This DataLoader supports keeping PatientLink available in the TestResults query but avoids the N+1 query issue that it previously had from the TestOrder join